docker-compose.yml

```diff
@@ -318,6 +318,8 @@ services:
     environment:
       TAG:
       PULUMI_CONFIG_PASSPHRASE: local-grapl-passphrase
+      # This gets exported to True by `make test-integration`
+      SHOULD_DEPLOY_INTEGRATION_TESTS: "${SHOULD_DEPLOY_INTEGRATION_TESTS:-False}"
```

Comment:
This here is probably the most debatable aspect of this PR - we are _optionally deploying resources_ based on an environment variable.

Happy to change it if people have any better ideas.